package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "electrode-static-paths",
-  "version": "1.1.0",
+  "version": "2.0.0",
```

Comment:
please don't manually update version in `package.json`.

Reply:
vs what?